### PR TITLE
Add survey comment to new issues.

### PR DESCRIPTION
**Pull request by tadelesh**
Add survey link to every issue and try to gather user feedback in a more active way.

### CODE REVIEW

**Comment by xboxeer** [comment]
Add some context, I'm recently help dealing with dotnet SDK issues and once an issue is closed, I'll advise customer to fill a survey and tell us their over experience of our SDK, the response rate is higher than expect and already have customer willing to talk with us, so it is worth to do the same in Golang SDK since we lack external customer study.

File: .github/fabricbot.json [hunk 1]
```diff
@@ -50,6 +50,12 @@
             "parameters": {
               "label": "needs-triage"
             }
+          },
+          {
+            "name": "addReply",
+            "parameters": {
+              "comment": "Hi @${issueAuthor}. Thank you for your feedback and we will look into it soon. Meanwhile, feel free to share your experience using the Azure SDK in this [survey](https://www.surveymonkey.com/r/FWPGFGG)."
```
Comment (RickWinter) [inline .github/fabricbot.json:57]:
```suggestion
              "comment": "Hi @${issueAuthor}. Thank you for your feedback and we will look into it soon. Meanwhile, feel free to share your experience using the Azure SDK in this [survey](https://aka.ms/azsdk/go/issuesurvey)."
```

**Review by RickWinter — CHANGES_REQUESTED** [review]

**Comment by RickWinter** [comment]
This change seems reasonable.  I've switched it to an aka.ms link (already live) so that we can quickly change it everywhere should that necessity arise.

@sandeep-sen , @mikekistler Can you review the questions

**Review by mikekistler — APPROVED** [review]
Looks good! 👍

**Review by xboxeer — APPROVED** [review]

**Review by lirenhe — APPROVED** [review]

**Review by xboxeer — APPROVED** [review]

**Review by RickWinter — APPROVED** [review]

**Pull request merged**